evm/src/fuzz/invariant/executor.rs

```diff
@@ -230,23 +239,28 @@ impl<'a> InvariantExecutor<'a> {
         }
 
         // Stores fuzz state for use with [fuzz_calldata_from_state].
-        let fuzz_state: EvmFuzzState = build_initial_state(self.executor.backend().mem_db());
+        let fuzz_state: EvmFuzzState =
+            build_initial_state(self.executor.backend().mem_db(), self.config.include_storage);
```

Comment:
just leaving a note, that if this gets merged https://github.com/foundry-rs/foundry/pull/2929 , we should update `build_initial_state` to reflect that